site/en/install/source_windows.md

```diff
@@ -275,7 +275,7 @@ For GPU support, add the CUDA and cuDNN bin directories to your `$PATH`:
 </pre>
 
 
-## Tested build configurations
+##<a name=tested_build_configurations></a> Tested build configurations
```

Comment:
Move before heading (and wrap in quotes, please)

Reply:
@lamberta , sorry for the delay. I missed the notification. I have made the requested changes now. Please see if we are good to go!